docs/src/main/sphinx/installation/cli.rst

```diff
@@ -37,7 +38,10 @@ the version:
 
     java -jar trino-cli-*-executable.jar --version
 
-The syntax can be used for the examples in the following sections.
+The syntax can be used for the examples in the following sections. In addition,
+using the ``java`` command allows you to add configuration options for the Java
+runtime with the ``-D`` syntax. You can use this for debugging and
+troubleshooting, for example :ref:`cli-kerberos-debug`.
```

Comment:
Two things here. 
1. Replace "for example" with "such as for".  But see point 2 first.
"for example" is not a conjunction like "but" that can join two parts of a sentence. It should always begin a sentence or clause.
2. As I remember, this :ref: form will take the wording of the following title and place it here. That will place an odd init-cap "Additional" in the middle of a sentence here. Should this be something like:

troubleshooting, such as when :ref:` specifying additional kerberos debug options <cli-kerberos-debug`.

(I haven't built this doc to see whether I got that syntax right.)